btcstaking-tracker/stakingeventwatcher/tracked_delegations.go

```diff
@@ -86,3 +96,28 @@ func (td *TrackedDelegations) RemoveDelegation(stakingTxHash chainhash.Hash) {
 
 	delete(td.mapping, stakingTxHash)
 }
+
+func (td *TrackedDelegations) HasDelegationChanged(
+	stakingTxHash chainhash.Hash,
+	newDelegation *newDelegation,
+) (bool, bool) {
```

Comment:
suggestion to name returns 
```suggestion
) (exists bool, statusUpdated bool) {
```